burro/config/defaults.ini

```diff
@@ -29,7 +29,9 @@ drift_gain = 0.15
 
 [ackermann_car_navio]
 throttle_channel = 5
+throttle_channel_invert = False
 steering_channel = 3
+steering_channel_invert = True
```

Comment:
It'd be better to have both this as the throttle invert options as members of the car category, so that they can have effect on other types of vehicles as well.